addons/pt/saft/invoices.go

```diff
@@ -26,12 +30,18 @@ var (
 	codeRegexp     = regexp.MustCompile(codePattern)
 )
 
+// invoiceWorkTypes is the list of work document types that map from GOBL invoices
 var invoiceWorkTypes = []cbc.Code{
 	WorkTypeProforma,
 	WorkTypeConsignmentInv,
 	WorkTypeConsignmentCredit,
 }
 
+// simplifiedMaxAmount is the maximum amount allowed for simplified invoices according to
+// Article 40 of the Portuguese VAT Code. This is the most permissive limit (for retail
+// sales) since we cannot distinguish between retail and non-retail sales.
+var simplifiedMaxAmount = num.MakeAmount(1000, 0)
```

Comment:
I've always tried to avoid these types of validations in GOBL that may change over time or have specific use cases, without a specific structure to deal with them like tax rates.

Appreciating that this is something that is being requested, I wonder if it could be handled by "scenarios" with some extra tweeks, at least then tags could be used to distinguish between retail and non-retail scenarios perhaps.

Reply:
In fact, in the last round of feedback from the certifiers, they complained that we hadn't apply the 100 eur limit. Let's discuss whether it's worth distinguishing between retail and non-retail and how to better enforce this constraint (via validations or scenarios).